docs/topics/auth/customizing.txt

```diff
@@ -299,6 +299,32 @@ example, the following checks if a user may view tasks::
 
     user.has_perm('app.view_task')
 
+.. versionadded:: 1.11
```

Comment:
As above, use a note after the section rather than putting a versionadded annotation in the middle of a section without a clear designation of what it applies to.